java/vector/src/main/java/org/apache/arrow/vector/compare/RangeEqualsVisitor.java

```diff
@@ -170,6 +171,11 @@ public Boolean visit(UnionVector left, Range range) {
     return compareUnionVectors(range);
   }
 
+  @Override
+  public Boolean visit(DenseUnionVector left, Range range) {
+    throw new UnsupportedOperationException();
```

Comment:
Open a JIRA to fill this and add a comment here?

Reply:
Sure. Will create one.